src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public String getRandomBall() {

```

Comment:
Then create a Lottery class with getRandomBall() method, which will return the ball. This method should create a ball with a random color and a random number (the maximum possible number should be 100) and return this ball from the method.
```suggestion
    public Ball getRandomBall() {

```